src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 0 2px;
+  border-radius: 0.5px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:first-child,
```

Comment:
For consistency, better to change the selector for the `stars--1` modifier to `.stars--1 .stars__star:nth-child(-n+1)`.